lib/common.js

```diff
@@ -282,10 +282,10 @@ const getFieldsCount = (fields, version = CURRENT_DD_VERSION) => {
   };
 };
 
-const getResourcesCount = fields => {
+const getResourcesCount = (fields, version = CURRENT_DD_VERSION) => {
```

Comment:
Similar. The version fallback should be done in the function and not by the caller.